src/opendr/perception/object_detection_2d/nanodet/algorithm/nanodet/model/fpn/pan.py

```diff
@@ -84,11 +88,10 @@ def forward(self, inputs):
 
         # part 2: add bottom-up path
         for i in range(0, used_backbone_levels - 1):
-            inter_outs[i + 1] += F.interpolate(
+            inter_outs[i + 1] = inter_outs[i + 1] + F.interpolate(
```

Comment:
```suggestion
            inter_outs[i + 1] += F.interpolate(
```

Reply:
In Onnx with pytorch documentation it has a comment to Avoid in-place operations in some scenarios. I am not sure if it is ok in this particular senario so I change it.

Reply:
If that's so then ignore my suggestion